app/config/api_version

```diff
@@ -1 +1 @@
-master
+VIMC-3062
```

Comment:
Once https://github.com/vimc/montagu-api/pull/403 merged this should be returned to master